MapboxNavigation/DayStyle.swift

```diff
@@ -91,6 +91,10 @@ open class DayStyle: Style {
         ExitView.appearance().foregroundColor = .black
         FloatingButton.appearance().backgroundColor = #colorLiteral(red: 1, green: 1, blue: 1, alpha: 1)
         FloatingButton.appearance().tintColor = tintColor
+        GenericRouteShield.appearance().backgroundColor = .clear
+        GenericRouteShield.appearance().borderWidth = 1.0
+        GenericRouteShield.appearance().cornerRadius = 5.0
+        GenericRouteShield.appearance().foregroundColor = .black
```

Comment:
How does this look for the night style?

Reply:
Pretty good, all we end up having to do is needing to change the `foregroundColor`: [Line](https://github.com/mapbox/mapbox-navigation-ios/pull/1417/files/e578eb00dfe627348cb826502f9b98885a6e7260#diff-f33b72188b8d837da672df7b12e4b6c6R200) 

![Example](https://user-images.githubusercontent.com/887225/39900487-6afa9182-547f-11e8-8b87-f3cd8fbfc2c6.png)

Reply:
Oh i missed that, 👍